torchvision/transforms/transforms.py

```diff
@@ -46,6 +46,7 @@
     "RandomPerspective",
     "RandomErasing",
     "GaussianBlur",
+    "GaussianNoise",
```

Comment:
You also need to add into docs. 

Edit 

https://github.com/pytorch/vision/blob/main/docs/source/transforms.rst  and

https://github.com/pytorch/vision/blob/main/gallery/plot_transforms.py#L121

Reply:
Yes will do.